projects/mtg/include/MTGRules.h

```diff
@@ -58,6 +58,10 @@ class MTGEventBonus: public PermanentAbility
     int Angel[2];
     bool dragonbonusgranted[2];
     int dragon[2];
+	bool eldrazibonusgranted[2];
+	int eldrazi[2];
+	bool werewolfbonusgranted[2];
+	int werewolf[2];
```

Comment:
ok set them to those settings, :) corssing fingers and hoping it doesnt turn out wrong. what I dont get is why does it look fantastic in the compiler, but then in commit looks like garbage?